src/com/hms_networks/americas/sc/thingworx/TWConnectorMain.java

```diff
@@ -216,7 +216,14 @@ public static void main(String[] args) {
         "Starting " + TWConnectorConsts.CONNECTOR_NAME + " " + TWConnectorConsts.CONNECTOR_VERSION);
 
     // Calculate local time offset and configure queue
-    TWTimeOffsetCalculator.calculateTimeOffsetMilliseconds();
+    try {
+      TWTimeOffsetCalculator.calculateTimeOffsetMilliseconds(
+          connectorConfig.getFtpUser(), connectorConfig.getFtpPassword());
+    } catch (JSONException e) {
+      Logger.LOG_WARN(
+          "An issue occured retrieving FTP user credentials from the configuration file.");
```

Comment:
"issue" should be changed to "error". Maybe this is more of a personal preference, but "issue" sounds too informal, and is also inconsistent with how we typically say "error"